sample/src/main/java/com/polidea/cockpit/sample/VariantIndependentBaseMainActivity.kt

```diff
@@ -19,8 +19,8 @@ abstract class VariantIndependentBaseMainActivity : AppCompatActivity() {
         setContentView(R.layout.activity_main)
         initViews()
 
-        Cockpit.setOnFontSizeChangeListener(
-                object : CockpitParamChangeListener<Int> {
+        Cockpit.addOnFontSizeChangeListener(
```

Comment:
If you write `PropertyChangeListener` in Java, this code could be simplified to:
```
Cockpit.addOnFontSizeChangeListener { oldValue, newValue ->
            Toast.makeText(this@MainActivity,
                    "fontSize changed, old value = $oldValue, new value = $newValue",
                    Toast.LENGTH_SHORT).show()
        }
```